core/src/main/java/com/linecorp/armeria/client/HttpChannelPool.java

```diff
@@ -276,7 +285,15 @@ private boolean usePendingAcquisition(SessionProtocol desiredProtocol, PoolKey k
             if (cause == null) {
                 final SessionProtocol actualProtocol = pch.protocol();
                 if (actualProtocol.isMultiplex()) {
-                    promise.complete(pch);
+                    final HttpSession session = HttpSession.get(pch.get());
+                    logger.info("session.unfinishedResponses(): {}, session.maxUnfinishedResponses(): {}",
+                                session.unfinishedResponses(), session.maxUnfinishedResponses());
```

Comment:
Probably `debug` is better? Or do you intend to remove it before removing draft?